src/ActionList/List.tsx

```diff
@@ -49,25 +67,34 @@ export const List = React.forwardRef<HTMLUListElement, ActionListProps>(
       selectionVariant: containerSelectionVariant, // TODO: Remove after DropdownMenu2 deprecation
     } = React.useContext(ActionListContainerContext)
 
+    const id = useId()
+    if (headingProps) {
+      headingProps.id = id
+    }
+
     return (
-      <ListBox
-        sx={merge(styles, sxProp as SxProp)}
-        role={role || listRole}
-        aria-labelledby={listLabelledBy}
-        {...props}
-        ref={forwardedRef}
-      >
-        <ListContext.Provider
-          value={{
-            variant,
-            selectionVariant: selectionVariant || containerSelectionVariant,
-            showDividers,
-            role: role || listRole,
-          }}
+      <Box sx={merge(outerStyles, sxProp as SxProp)}>
+        {headingProps && <Heading {...headingProps} />}
+        <ListBox
+          sx={innerStyles}
+          role={role || listRole}
+          aria-labelledby={headingProps ? id : listLabelledBy}
+          aria-busy="true"
```

Comment:
Curious about this aria-busy, why is that needed for all lists?

Reply:
It's not needed for all lists. It fixes a violation in the Multiple Sections ActionMenu story. To my surprise, that value passes down just fine through the `ActionList` component, so I'm passing it in that story instead now.